packages/playback-core/src/request-errors.ts

```diff
@@ -0,0 +1,207 @@
+        mediaError.errorCategory = category;
+        mediaError.muxCode = MuxErrorCode.TOKEN_MALFORMED;
+        return mediaError;
+      }
+      if (isJWTSubMismatch(jwtObj, playbackId)) {
+        const message = i18n(
+          `The video’s playback ID does not match the one encoded in the ${tokenNamePrefix}-token.`,
+          translate
+        );
+        const context = i18n(
+          `Specified playback ID: {playbackId} and the playback ID encoded in the ${tokenNamePrefix}-token: {tokenPlaybackId}`,
+          translate
+        ).format({
+          playbackId,
+          tokenPlaybackId: jwtObj.sub,
+        });
+        const mediaError = new MediaError(message, mediaErrorCode, true, context);
+        mediaError.errorCategory = category;
+        mediaError.muxCode = MuxErrorCode.TOKEN_SUB_MISMATCH;
+        return mediaError;
+      }
+      /** @TODO aud mismatches are 403 for video URL responses but 400 for DRM. Should change this for consistency (CJP) */
+      if (isJWTAudMissing(jwtObj, expectedAud)) {
+        const message = i18n(`The ${tokenNamePrefix}-token is formatted with incorrect information.`, translate);
+        const context = i18n(
+          `The ${tokenNamePrefix}-token has no aud value. aud value should be ${expectedAud}.`,
+          translate
+        );
+        const mediaError = new MediaError(message, mediaErrorCode, true, context);
+        mediaError.errorCategory = category;
+        mediaError.muxCode = MuxErrorCode.TOKEN_AUD_MISSING;
+        return mediaError;
+      }
+      /** @TODO aud mismatches are 403 for video URL responses but 400 for DRM. Should change this for consistency (CJP) */
+      if (isJWTAudMismatch(jwtObj, expectedAud)) {
+        const message = i18n(`The ${tokenNamePrefix}-token is formatted with incorrect information.`, translate);
+        const context = i18n(
+          `The ${tokenNamePrefix}-token has an incorrect aud value: {tokenType}. aud value should be ${expectedAud}.`,
+          translate
+        ).format({
+          tokenType: jwtObj.aud,
+        });
+        const mediaError = new MediaError(message, mediaErrorCode, true, context);
+        mediaError.errorCategory = category;
+        mediaError.muxCode = MuxErrorCode.TOKEN_AUD_MISMATCH;
+        return mediaError;
+      }
+
+      /**
+       * @TODO invalid playback URLs and playback IDs are 404 for video URL responses but 400 for DRM. Should change this for consistency (CJP)
+       */
+
+      /** @TODO omitted tokens are 403 for video URL responses but 400 for DRM. Should change this for consistency (CJP) */
+      // NOTE: This *should* not happen for DRM, since the drm token
+      // is currently used to detect whether or not DRM should
+      // be setup at all. Including for exhaustiveness. (CJP)
+    } else {
+      const message = i18n(
+        `${status ?? 403} error trying to access this ${category} URL. If this is a signed URL, you might need to provide a ${category} token.`,
```

Comment:
NOTE: These HTTP statuses should be normalized in prod soon, but we should still potentially consider updating our default message to not include `403` and instead replace it with something like `"Authorization"`